LinearDecayCorrelationModel.cpp

```diff
@@ -21,18 +21,22 @@
 //     20-Dec-2012   JPK      Fixed bug in constructor.
 //     18-Feb-2013   JPK      Removed getNumCorrelationParameterGroups,
 //                            now provided on base class.
-//     03-Sep-2021   SCM      Removed IRIX support.
```

Comment:
Undo reversion of this change (like the `#ifdef` below)

Reply:
Fixed